libpod/runtime.go

```diff
@@ -210,13 +215,14 @@ var (
 			"/usr/libexec/crio/conmon",
 			"/usr/local/lib/podman/conmon",
 			"/usr/local/libexec/crio/conmon",
-			"/usr/bin/conmon",
+			"/usr/binWe could show the hard-coded default of the runtime bu/conmon",
```

Comment:
Is this intentional?

Reply:
Sorry for missing that :( No intention at all, but my new keyboard keeps pasting stuff in vim...